sklearn/utils/extmath.py

```diff
@@ -682,7 +730,14 @@ def _randomized_eigsh(
     """
     if selection == "value":  # pragma: no cover
         # to do : an algorithm can be found in the Halko et al reference
```

Comment:
Remove the todo, possibly replace it with a short explanation such as `# Call Halko et al 5.6 randomized eigs solver`